src/DetailsView/components/static-content-common.scss

```diff
@@ -10,7 +10,8 @@
     margin-bottom: auto;
 
     ol {
-        -webkit-padding-start: 16px;
```

Comment:
Is the `static-content-in-details-view` style actually used? I couldn't find it in a search of the repo.

Reply:
I do see the class used in `src\DetailsView\components\static-content-details-view.tsx` and `src\DetailsView\components\target-page-changed-view.tsx`, however it's not clear to me if the nested `ol` is in use. 

I do see some interesting font-size and style inconsistencies when the target page changes. I'll plan to look into this closer in another pull request. I'm imagining there's an opportunity to consolidate styles here to prevent the style shift when the target page changes.

The screenshot below shows the "Needs review" panel:

![](https://user-images.githubusercontent.com/2180540/211046637-f1f43f22-78bb-40e6-88d6-93589325b2e5.png)

The next screenshot shows the same "Needs review" panel, when the target page changes. There is a change to the font-size and style, especially on the italicized "possible":
![](https://user-images.githubusercontent.com/2180540/211046835-119fb055-e6a6-4ed3-9b89-5f4968aa6cc0.png)